src/Core/Services/Implementations/UserService.cs

```diff
@@ -288,6 +272,63 @@ public async Task<IdentityResult> DeleteAsync(User user, string token)
         return await DeleteAsync(user);
     }
 
+#nullable enable
+    private async Task<IEnumerable<(Guid UserId, string? ErrorMessage)>> ValidateDeleteUsersAsync(List<User> users)
+    {
+        var userResult = new List<(Guid UserId, string? ErrorMessage)>();
+
+        foreach (var user in users)
+        {
+            // Check if user is the only owner of any organizations.
+            var onlyOwnerCount = await _organizationUserRepository.GetCountByOnlyOwnerAsync(user.Id);
+            if (onlyOwnerCount > 0)
+            {
+                var deletedOrg = false;
+                var orgs = await _organizationUserRepository.GetManyDetailsByUserAsync(user.Id,
+                    OrganizationUserStatusType.Confirmed);
+                if (orgs.Count == 1)
```

Comment:
For example: if we're deleting a user, and they're the sole owner of an organization, then they must belong to at least 2 organizations (our org + the org they own), so this will always evaluate to false. I don't think this would do any harm, but as I said, it's not a great fit.